modules/grizzly/src/main/java/org/glassfish/grizzly/threadpool/DefaultWorkerThread.java

```diff
@@ -104,4 +104,8 @@ public void setTransactionTimeout(long timeout, TimeUnit timeunit) {
         this.transactionTimeoutMillis =
                 TimeUnit.MILLISECONDS.convert(timeout, timeunit);
     }
+
+    @Override
+    public void destroy() {
```

Comment:
or remove from the interface, see https://github.com/eclipse-ee4j/grizzly/pull/2055

Reply:
Yes, if there's no calls to destroy() in Grizzly, it would probably be better to
remove it from the interface.  (Or remove any calls, which clearly have no effect.)

Reply:
OK I'll check whether it can be removed from the interface I assume that #2055 passes all tests?

Reply:
there are some test failing, But I guess it's unrelated:

```
INFO: Stopped listener bound to [0.0.0.0:18901]
[ERROR] Tests run: 9, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.745 s <<< FAILURE! - in org.glassfish.grizzly.servlet.HttpServerTest
[ERROR] testStartSecureWithConfiguration(org.glassfish.grizzly.servlet.HttpServerTest)  Time elapsed: 0.433 s  <<< ERROR!
javax.net.ssl.SSLHandshakeException: Remote host closed connection during handshake
	at org.glassfish.grizzly.servlet.HttpServerTest.testStartSecureWithConfiguration(HttpServerTest.java:256)
Caused by: java.io.EOFException: SSL peer shut down incorrectly
	at org.glassfish.grizzly.servlet.HttpServerTest.testStartSecureWithConfiguration(HttpServerTest.java:256)
```

Reply:
If that's JDK 11 there are SSL failures when building on JDK 11. That's the next thing to address.